backpack/extensions/firstorder/batch_grad/batch_grad_base.py

```diff
@@ -1,17 +1,40 @@
+"""Passes the calls for the parameters to the derivatives class."""
 from backpack.extensions.firstorder.base import FirstOrderModuleExtension
 
 
 class BatchGradBase(FirstOrderModuleExtension):
+    """Passes the calls for the parameters to the derivatives class.
+
+    This class implements functions with method names from params.
+
+    If child class wants to overwrite these methods
+    - for example to support an additional external module -
+    it can do so using the interface for parameter "param1"
+    param1(ext, module, g_inp, g_out, bpQuantities):
+        return batch_grads
```

Comment:
Apply code-style formatting

Reply:
I added a line before and after. I didn't find out how to add multiline code.

Reply:
Maybe something along [these lines](https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e#file-docstrings-py-L270-L274) works?